cmd/utils/cmd.go

```diff
@@ -85,6 +92,19 @@ func StartNode(stack *node.Node) {
 	}()
 }
 
+func monitorFreeDiskSpace(path string) {
+	for {
+		freeSpace := getFreeDiskSpace(path)
+		if freeSpace < freeDiskSpaceCritical {
+			log.Error("Low disk space. Gracefully shutting down Geth to prevent database corruption.", "available", freeSpace/1024/1024)
+			sigtermCh <- syscall.SIGTERM
```

Comment:
IIUC: Once this hits, it will hit again every `5` seconds, and after `50` seconds the node will `panic`. So you should exit the loop after sending the `SIGTERM`

Reply:
fixed